internal/provider/resource_local_sensitive_file_test.go

```diff
@@ -0,0 +1,216 @@
+package provider
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	r "github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestLocalSensitiveFile_Basic(t *testing.T) {
+	td := createTestingTempDir(t)
+	defer os.RemoveAll(td)
+
+	f := filepath.Join(td, "local_file")
+	f = strings.ReplaceAll(f, `\`, `\\`)
+
+	var cases = []struct {
+		path    string
+		content string
+		config  string
+	}{
+		{
+			f,
+			"This is some content", fmt.Sprintf(`
+				resource "local_file" "file" {
+				  content  = "This is some content"
+				  filename = "%s"
+				}`, f,
+			),
+		},
+		{
+			f,
+			"This is some sensitive content", fmt.Sprintf(`
+				resource "local_file" "file" {
```

Comment:
And this should be local_sensitive_file with attribute content instead of sensitive_content, correct?